app/views/state_file/questions/md_tax_refund/_md_bank_details.html.erb

```diff
@@ -2,26 +2,6 @@
   <p class="spacing-below-0"><%= t("state_file.questions.tax_refund.bank_details.bank_title") %></p>
   <p class="text--small"><%= t("state_file.questions.tax_refund.bank_details.foreign_accounts") %></p>
   <div>
-    <% if owe_taxes %>
```

Comment:
this bank details partial is only used for the md_tax_refund page, maryland uses the general taxes_owed page and so will use the other bank details partial